cli/azd/pkg/exec/command_runner.go

```diff
@@ -125,31 +125,20 @@ func (r *commandRunner) Run(ctx context.Context, args RunArgs) (RunResult, error
 		}
 	}
 
-	logTitle := strings.Builder{}
-	logBody := strings.Builder{}
-	defer func() {
-		logTitle.WriteString(logBody.String())
-		log.Print(logTitle.String())
-	}()
-
-	logTitle.WriteString(fmt.Sprintf("Run exec: '%s %s' ",
-		args.Cmd,
-		RedactSensitiveData(
-			strings.Join(RedactSensitiveArgs(args.Args, args.SensitiveData), " "))))
-
-	debugLogEnabled := r.debugLogging
+	debugLogging := r.debugLogging
 	if args.DebugLogging != nil {
-		debugLogEnabled = *args.DebugLogging
+		debugLogging = *args.DebugLogging
 	}
 
-	if debugLogEnabled && len(args.Env) > 0 {
-		logBody.WriteString("Additional env:\n")
-		for _, kv := range args.Env {
-			logBody.WriteString(fmt.Sprintf("   %s\n", kv))
-		}
-	}
+	logMsg := logBuilder{}
+	defer func() {
+		logMsg.Write(debugLogging, args.SensitiveData)
+	}()
+	logMsg.args = append([]string{args.Cmd}, args.Args...)
+	logMsg.env = args.Env
```

Comment:
any reason not to set the `args` and `env` in the initialize struct call

Reply:
done